kmip/pie/client.py

```diff
@@ -541,6 +542,59 @@ def activate(self, uid=None):
             message = result.result_message.value
             raise exceptions.KmipOperationFailure(status, reason, message)
 
+    def revoke(self, code, uid=None, message=None,
+               compromise_occurrence_date=None):
+        """
+        Revoke a managed object stored by a KMIP appliance.
+
+        Args:
+            code (RevocationReasonCode): An enumeration indicating the
+                revocation reason.
+            uid (string): The unique ID of the managed object to revoke.
+            message (string): A message regarding the revocation.
+            compromise_occurrence_date (DateTime): A Date-Time primitive
+                indicating when the managed object was firstly believed
+                to be compromised.
```

Comment:
Let's make `compromise_occurrence_date` a Python integer, something a user/dev could easily get by using `int(time.time())`. Internally the client should use this value to create a `DateTime` for the the `KMIPProxy`.